LambdaEngine/Source/Rendering/RenderGraph.cpp

```diff
@@ -2168,6 +2179,21 @@ namespace LambdaEngine
 
 				pPipelineStage->ppGraphicsCommandLists[f]		= m_pGraphicsDevice->CreateCommandList(pPipelineStage->ppGraphicsCommandAllocators[f], &graphicsCommandListDesc);
 
+				Profiler::GetGPUProfiler()->AddTimestamp(pPipelineStage->ppGraphicsCommandLists[f]);
```

Comment:
It would be nice to be able to add Names to timestamps

Reply:
Names added on local